packages/contracts-bedrock/test/setup/Events.sol

```diff
@@ -105,7 +105,7 @@ contract Events {
 
     event Paused(string identifier);
 
-    event Unpaused();
+    event Unpaused(string identifier);
```

Comment:
TODO: We will need to update monitoring accordingly

Reply:
Is that tracked?